source/jormungandr/tests/journey_common_tests.py

```diff
@@ -34,7 +34,8 @@
 from jormungandr import i_manager
 from .check_utils import *
 import mock
-import  os
+import os
```

Comment:
module is not used

Reply:
yes it is : 
https://github.com/CanalTP/navitia/pull/2467/files/91d33040c7d35e4a512683688f785f24818272a0#diff-0832703183fb3233e60448ac96ec726bR931

Reply:
ok!